index.html

```diff
@@ -21,6 +21,124 @@ <h1 class="hero">Irotori Baroka</h1>
       </p>
       <button type="button" class="header-button">Start collaboration</button>
     </section>
-
+    <div class="vector-img"></div>
+    <section id="projects">
+      <header>
+        <h2>Projects</h2>
+      </header>
```

Comment:
- [ ] [OPTIONAL] Kindly use the same spacing as the one you used for the About title.
- Project title
<img width="363" alt="Screen Shot 2022-03-17 at 14 50 08" src="https://user-images.githubusercontent.com/28307960/158812468-85c7c851-f4f0-463e-ac8e-a011bc8095b6.png">
- About title
<img width="357" alt="Screen Shot 2022-03-17 at 14 51 31" src="https://user-images.githubusercontent.com/28307960/158812539-30190639-1864-422f-b005-48e38bbf9da4.png">